### PR TITLE
feat: Build the header with the language for the public sections

### PR DESCRIPTION
**Pull request by GusBaamonde**
- Build the header with the language for the public sections. Sign up, forgot and login.
- Add the images of the flags.
- Add the script to select the flag and show how it works.
- Change the header phrase. Now it is part of the content.

https://cdn.fromdoppler.com/doppler-ui-library/build182/login.html

header_language

### CODE REVIEW

**Comment by andresmoschini** [comment]
> I'm not sure if it's correct that this language selector is in Signup.

No entendí

**Comment by IgnacioRodrigues** [comment]
The styles of the language selector should be in the base so that it is a reusable module and not in the signup

File: assets/js/main.js [hunk 1]
```diff
@@ -69,3 +69,14 @@ $(".show-hide").click(function() {
...
+  // languague
+  $('.language-selector .option').click(function(){
+    $(this).toggleClass('open');
+  });
+
+  $(window).click(function(e) {
+    if ($('.language-selector').has(e.target).length == 0 && !$('.language-selector').is(e.target)) {
+      $('.language-selector .option').removeClass('open');
+    }
+  });
```
Comment (andresmoschini) [inline assets/js/main.js]: Could you rewrite these scripts in order to make them work in any place (aka WebApp) as the previous ones? probably @fcoronelmakingsense or @damianmuti could help you with that.

Comment (fcoronelmakingsense) [reply assets/js/main.js]: `$("body").on('click', '.language-selector .option', function() {
  $('.language-selector .option').toggleClass('open');
});`

Comment (fcoronelmakingsense) [reply assets/js/main.js]: `$(document).on('mouseup', function(e){
  var menu = $(".user-menu");
  var languageSelector = $(".language-selector");

  if (!menu.is(e.target) && menu.has(e.target).length === 0){
    menu.removeClass('open');
  }

  if (!languageSelector.is(e.target) && languageSelector.has(e.target).length === 0){
    $('.language-selector .option').removeClass('open');
  }
});`

**Comment by GusBaamonde** [comment]
create the module _language.scss thanks Nachou!

File: assets/js/main.js [hunk 2]
```diff
@@ -69,3 +79,14 @@ $(".show-hide").click(function() {
...
+  // languague
+  // $('.language-selector .option').click(function(){
+  //   $(this).toggleClass('open');
+  // });
+
+  // $(window).click(function(e) {
+  //   if ($('.language-selector').has(e.target).length == 0 && !$('.language-selector').is(e.target)) {
+  //     $('.language-selector .option').removeClass('open');
+  //   }
+  // });
```
Comment (andresmoschini) [inline assets/js/main.js]: Could you remove commented code?

**Comment by andresmoschini** [comment]
@andresmoschini will this require any change in webapp side if we do not include the language selection yet?

File: assets/js/main.js (hunk 2 above)
Comment (GusBaamonde) [reply assets/js/main.js]: Sorry, I thought I had deleted the comments

**Comment by GusBaamonde** [comment]
> @andresmoschini will this require any change in webapp side if we do not include the language selection yet?

Yes @andresmoschini  If you must change the header, even if we do not use the language now. And place the header phrase within the content.

**Pull request merged**